artifacts/operator-configs/controller-conf.yaml

```diff
@@ -37,6 +37,7 @@ data:
   apim-deployment-readinessProbe-initialDelaySeconds: "120"
   apim-deployment-readinessProbe-periodSeconds: "10"
   apim-deployment-readinessProbe-failureThreshold: "3"
+  apim-deployment-securityContext-runAsUser: "802"
```

Comment:
This is wrong. Lets consider this config as follows.

`apim-deployment-securityContext: "runAsUser: 802"`

In this case we are able to give any config to the security context